packages/api/src/apps/api/app.ts

```diff
@@ -238,6 +229,19 @@ export default async (env: ProcessEnv) => {
     await services.globalStats.update();
   }
 
+  // wait update rate before running loops, since all state was just updated on init
+  await new Promise((res) => setTimeout(res, updateRateS * 1000));
```

Comment:
previously we would do the state updates on init, then go into the loops which would immediately run the updates again. this pauses the loops until some time has passed before re-running updates. This is not part of the bugfix, but an unrelated improvement